### PR TITLE
consistent usage of checkinstall

### PR DESCRIPTION
**Pull request by stemd**
There is no sense to install two libs with make install and
other two with checkinstall.

On debian based distros usage of deb packages and tool like checkinstall
is preferred, as manually installing something in /usr/local and later
on upgrade (which will surely happen once)
installing that same package in location like /usr/bin is
a sure way to cause headache until you try using something like:
which binary_name
and then realizing that you have two installs, and you totally
forgot that you four of more years ago tried something and ...

In short, make install isn't BAD, but checkinstall is better.

I know, this is a bit long commit msg, but this is experience
talking, not me. :)

---
This change is [Reviewable](https://reviewable.io/reviews/grayhatter/utox/249)

### CODE REVIEW

**Comment by stemd** [comment]
As above, after correcting docs/BUILD.md build instructions for debian/Ubuntu,
I also added Croatian translation.

Have some more ideas (UI is rudimentary, needs polishing), after I check how it behaves on Windows
will probably try to make it behave more nice. Don't mind me, for alpha this is quite fine.

**Comment by GrayHatter** [comment]
can you please split these into two separate pull requests?

**Comment by stemd** [comment]
I'll try. :)

**Comment by GrayHatter** [comment]
:lgtm:

---

Reviewed 3 of 4 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/grayhatter/utox/249#-:-KEUMMFthWZ-ONyFMtoi:1417088721)*

**Comment by GrayHatter** [comment]
@stal888 @tsudoko 

anyone else can sign off on this one too

**Comment by stal888** [comment]
Rubber-stamping this for you, @grayhatter
:lgtm:  

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/grayhatter/utox/249#-:-KEUPU9F1AgzKMtyJriD:18088496)*
